Objects/typeobject.c

```diff
@@ -9157,6 +9158,22 @@ releasebuffer_call_python(PyObject *self, Py_buffer *buffer)
     Py_DECREF(mv);
 }
 
+static void
+releasebuffer_call_python(PyObject *self, Py_buffer *buffer)
```

Comment:
Can you inline this in `releasebuffer_call_python_inner` itself? I am not a fan of such once used small functions.

Reply:
Sure. I didn't do that so I wouldn't have to change the return statements, but I can just add a goto error.